write-fonts/src/tables/variations/ivs_builder.rs

```diff
@@ -78,8 +78,15 @@ impl VariationStoreBuilder {
     ) -> TemporaryDeltaSetId {
         let mut delta_set = Vec::with_capacity(deltas.len());
         for (region, delta) in deltas {
+            let delta = delta.into();
+            // a delta of zero is the same as not including this region;
```

Comment:
this is incorrect. A delta of zero is _not_ the same as not including a region. Imagine a glyph defines three masters (or regions): ({wght:400}, 100) is the default, then there are ({wght:500}, 100) and ({wght:700}, 200). The deltas for these two are respectively [0, 100], but the 0 there cannot be simply skipped as if it was not there. It has the effect of anchoring that location (or restoring it back) to the default value. If it was not there, there would be a linear interpolation between ({wght:400}, 100) and ({wght:700}, 200).
Only when _all_ deltas in a delta set are 0 they can be treated the same as them being empty.

Reply:
I'll modify the test to show what I mean